content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -632,7 +632,7 @@ The following are all the logical operators that you can use in the `operator` f
 
 The following operators can only be used with `nodeAffinity`.
 
-|    Operator    |    Behaviour    |
+|    Operator    |    Behavior    |
```

Comment:
@anavalo   both the words are same noun Its depend on the Region to region like in In British English, you use “behaviour.” In American English, you use “behavior" .

So Here the word Behaviour is not required to be changes as we are using same in the entire docs.

Reply:
In this `md`, both nouns are used. The correction is to use the same noun consistently on this page.

Reply:
Yes I seen the word Behavior is used consistently in this page. So its look good to me.